requirements.txt

```diff
@@ -0,0 +1,21 @@
+# Data analysis and manipulation
+numpy~=1.21.5
+pandas~=1.3.5
+
+# Testing
+pytest~=7.3.1
+
+# Machine learning
+scikit-learn~=1.0.1
+lightgbm~=3.3.5
```

Comment:
Should lgbm, catboost, xgboost be prerequisits?

Reply:
Yeah, because we use this libraries in crossval._curve.py file